.github/workflows/ci.yml

```diff
@@ -39,6 +39,11 @@ jobs:
       with:
         file: ./modules/codegen/target/scala-${{ matrix.scala.bincompat }}/scoverage-report/scoverage.xml
         fail_ci_if_error: true
+        steps:
+        - id: set-scala-versions
+          run: echo "::set-output name=scala_versions::$(sbt 'print codegen/crossScalaVersions' | grep '^\* ' | sed 's/^\* //')"
```

Comment:
`grep | sed` can be replaced with `awk '/^\* /{ print $2 }'`if you are so inclined

Reply:
Ended up needing to do it in Scala anyway, but good call